pages/1.12/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -164,168 +184,106 @@ curl -i -k \
-   ```
-
-1. Compress the Docker credentials.
-
-   ```bash
-   sudo tar cvzf docker.tar.gz .docker
-   ```
-
-    **Tip:** You can confirm that the operation succeeded with this command.
-
-   ```bash
-   tar -tvf docker.tar.gz
-   ```
-
-   You should see output similar to this.
-
-   ```bash
-   drwxr-xr-x  0 user group       0 Jan 23 15:47 .docker/
-   -rw-------  0 user group     217 Jan 23 15:48 .docker/config.json
-   ```
-
-1. Copy the Docker credentials file to one of your masters with the public master IP address (`<public-master-ip>`) specified.
-
-   **Tip:** You can find the public master IP address by clicking on the cluster name in the top-left corner of the DC/OS GUI.
-
-   ```bash
-   scp docker.tar.gz core@<public-master-ip>:~
-   ```
-
-1. [SSH](/1.11/administering-clusters/sshcluster/) to the master node that contains the Docker credentials file.
-
-   ```bash
-   dcos node ssh --master-proxy --mesos-id=<master-id>
-   ```
-
-1. Store the IP address of each private agent in an environment variable. <!-- What is this step for -->The steps required depend on your [security mode](/1.11/security/ent/#security-modes).
-
-   ### Disabled
-
-   ```bash
-   PRIVATE_AGENT_IPS=$(curl -sS leader.mesos:5050/slaves | jq '.slaves[] | select(.reserved_resources.slave_public == null) | .hostname' | tr -d '"');
-   ```   
-
-   ### Permissive
-
-   ```bash
-   PRIVATE_AGENT_IPS=$(curl -sS leader.mesos:5050/slaves | jq '.slaves[] | select(.reserved_resources.slave_public == null) | .hostname' | tr -d '"');
-   ```
-
-   ### Strict
-
-   1.  Authenticate and save your authentication token to the `AUTH_TOKEN` environment variable, with your DC/OS username (`<username>`) and password (`<password>` specified.
 
-       ```bash
-       AUTH_TOKEN=$(curl -X POST localhost:8101/acs/api/v1/auth/login \
-       -d '{"uid":"<username>","password":"<password>"}' \
-       -H 'Content-Type: application/json' | jq -r '.token')
-       ```
-
-   1.  Download the certificate bundle, with your cluster URL (`<cluster-url>`) specified.
+# Step 4: Create a Service Account Secret
+In this step, a secret is created for the Marathon service account and stored in the Secret Store.
 
-       ```bash
-       sudo curl -k -v https://<cluster-url>/ca/dcos-ca.crt -o /etc/ssl/certs/dcos-ca.crt
-       ```
+  The secret created (`${SERVICE_ACCOUNT_SECRET}`) will contain the private key (`${PRIVATE_KEY}`) and the name of the service account (`${SERVICE_ACCOUNT_ID}`). This information will be used my Marathon to authenticate against DC/OS.
 
-   1.  Save the private IP address of each of your private agents to the `PRIVATE_AGENT_IPS` environment variable.
+  ## {.tabs}
 
-       ```bash
-       PRIVATE_AGENT_IPS=$(curl -sS --cacert /etc/ssl/certs/dcos-ca.crt \
-       -H "Authorization: token=$AUTH_TOKEN" https://leader.mesos:5050/slaves | jq '.slaves[] | select(.reserved_resources.slave_public == null) | .hostname' | tr -d '"');
-       ```
+  ### Disabled
```

Comment:
Disabled security mode is being completely removed in DC/OS 1.12. This section should be removed along with other references to _disabled_ mode in 1.12

Reply:
Good point 👍

Reply:
✅